Exercise_5.java

```diff
@@ -1,20 +1,36 @@
+//Could not solve Iterative quicksort. hoping to get some help during regular classes.
+

```

Comment:
This problem is not solved in regular class. The concept is to use stack to perform controlled recursion.